vulnerabilities/importers/apache_httpd.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and https://github.com/nexB/vulnerablecode/
+# The VulnerableCode software is licensed under the Apache License version 2.0.
+# Data generated with VulnerableCode require an acknowledgment.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+# When you publish or redistribute any data created with VulnerableCode or any VulnerableCode
+# derivative work, you must accompany this data with the following acknowledgment:
+#
+#  Generated with VulnerableCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
+#  OR CONDITIONS OF ANY KIND, either express or implied. No content created from
+#  VulnerableCode should be considered or used as legal advice. Consult an Attorney
+#  for any legal advice.
+#  VulnerableCode is a free software tool from nexB Inc. and others.
+#  Visit https://github.com/nexB/vulnerablecode/ for support and download.
+
+from dataclasses import dataclass
+import xml.etree.ElementTree as ET
```

Comment:
What's the benefit of  a shorter ET import?
IMHO for readability and clarity I would use `from import xml.etree import ElementTree` instead

Reply:
>What's the benefit of a shorter ET import? 

Nothing. 

>IMHO for readability and clarity I would use from import xml.etree import ElementTree instead


That works.

FWIW using the alias `ET` is a common practice. The docs used it https://docs.python.org/3/library/xml.etree.elementtree.html .

Reply:
Using `ElementTree`